<p>
	Isso porque os <em>RuntimeExceptions</em> ocorre por conta de problemas de lógicas do código do programador, que são situações excepcionais e não falhas em seu código, que é o que o compilador pega. Então o <em>Try/Catch</em> deve ser usado para recuperar de situações que não possam ser garantidas que serão bem-sucedidas, ou para mostrar uma mensagem e um rastreamento de pilha, a fim de que possa descobrir o que ocorreu.
</p>